frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -1203,7 +1203,6 @@ A custom wrapper around `fetch` that adds console-specific headers and allows fo
 | `method` | The HTTP method to use. Defaults to GET |
 | `options` | The options to pass to fetch |
 | `timeout` | The timeout in milliseconds |
-| `isEntireResponse` | The flag to control whether to return the entire content of the response or response body. The default is the response body. |
```

Comment:
here we are missing revert of two text changes which have been done in https://github.com/openshift/console/commit/0fb634f5e428c1b6b22544d01c63e86cf555fbc0